File: js/checkIfTitleString.js
// Problem# 219
// Check if a string title is a title string or not.
// A title string is one which has all the words in the string start with a upper case letter.
// https://edabit.com/challenge/6q9iTtQdnZZM5Cbai

const checkTitle = (title) =>
  title
    .split(" ")
    .map((word) => [...word][0])
    .every((letter) => letter === letter.toUpperCase());

module.exports = checkTitle;

File: python/check_if_title_string.py
"""
Check If It's a Title String

Check if a string txt is a title text or not. A title text is one which has all the words in the text start with an upper case letter.

https://edabit.com/challenge/CHwX2o6rqrBsL4gzr
"""


def check_title(txt: str) -> bool:
    return txt == txt.title()
